ClientApp/viking-vault/src/display_user_list/DeleteUserModal.tsx

```diff
@@ -4,8 +4,8 @@ import './DeleteUser.css';
 interface IDeleteUserModalProps{
     open: boolean;
     deletedUserName: string;
-    closeModal: any;
-    deleteUser: any;
+    closeModal: () => void;
+    deleteUser: () => void;
 }
 
 class DeleteUserModal extends React.Component<IDeleteUserModalProps, any>{ 
```

Comment:
If there is no state in this component, do we really need a class-based component?
A functional component would be more appropriate.

Take a look over the following: 

- [React documentation regarding functional components](https://reactjs.org/docs/components-and-props.html)
- [What are they | Why are they used](https://programmingwithmosh.com/react/react-functional-components/)